light-client/tests/integration.rs

```diff
@@ -0,0 +1,125 @@
+//! Light Client integration tests.
+//!
+//! These are all ignored by default, since they test against running
+//! `tendermint node --proxy_app=kvstore`. They can be run using:
+//!
+//! ```
+//! cargo test -- --ignored
+//! ```
+
+use tendermint_light_client::{
+    components::{
+        clock::SystemClock,
+        io::{AtHeight, Io, IoError, ProdIo},
+        scheduler,
+        verifier::ProdVerifier,
+    },
+    evidence::{Evidence, EvidenceReporter},
+    fork_detector::ProdForkDetector,
+    light_client::{self, LightClient},
+    peer_list::PeerList,
+    state::State,
+    store::{memory::MemoryStore, LightStore},
+    supervisor::{Handle, Instance, Supervisor},
+    types::{PeerId, Status, TrustThreshold},
+};
+
+use tendermint::abci::transaction::Hash as TransactionHash;
+
+use std::collections::HashMap;
+use std::time::Duration;
+
+fn make_instance(peer_id: PeerId, options: light_client::Options, io: ProdIo) -> Instance {
```

Comment:
I believe we have a similar method in the example, if this is common enough utility maybe it's time to lift it into the library.

Reply:
Yes, we plan to eventually deduplicate this type of code (#422), and clean up the tests utilities as well.

Reply:
Dope!